src/actions/approval.table.js

```diff
@@ -0,0 +1,93 @@
+import axios from 'axios';
+import { config } from 'dotenv';
+export const GET_TRIP_REQUESTS_SUCCESS = 'GET_TRIP_REQUESTS_SUCCESS';
+export const GET_TRIP_REQUESTS_FAIL = 'GET_TRIP_REQUESTS_FAIL';
+export const SET_SELLECTED_TRIP = 'SET_SELLECTED_TRIP';
+export const EDIT_TRIPREQUEST_STATUS_SUCCESS =
+	'EDIT_TRIPREQUEST_STATUS_SUCCESS';
+export const EDIT_TRIPREQUEST_STATUS_FAIL = 'EDIT_TRIPREQUEST_STATUS_FAIL';
+export const MAKE_OPEN_MODEL_TO_FALSE = 'MAKE_OPEN_MODEL_TO_FALSE';
+
+config();
+
+export const getTripRequests = props => async dispatch => {
```

Comment:
@shemaeric1234 Just in case this GET request takes longer, shouldn't you display a loader? This is applicable elsewhere you're making an HTTP request.